tests/bluetooth/tester/src/bttester.h

```diff
@@ -367,6 +367,12 @@ struct gap_pairing_consent_req_ev {
 	uint8_t address[6];
 } __packed;
 
+#define GAP_EV_BOND_LOST	0x8b
+struct gap_bond_lost_ev {
+	uint8_t address_type;
+	uint8_t address[6];
+} __packed;
```

Comment:
this is mainly due to historical reasons as initially we thought that this will be used also for BR/EDR

that said, we have some plans to cleanup BTP spec a bit and this could be good excuse to cleanup tester app too